other/upgrade_2-1_postgresql.sql

```diff
@@ -1261,6 +1262,67 @@ ALTER TABLE {$db_prefix}members
 	DROP IF EXISTS notify_announcements;
 ---#
 
+---# Upgrading auto notify setting
+---{
+$_GET['a'] = isset($_GET['a']) ? (int) $_GET['a'] : 0;
+$step_progress['name'] = 'Upgrading auto notify setting';
+$step_progress['current'] = $_GET['a'];
+
+$limit = 100000;
+$is_done = false;
+
+$request = $smcFunc['db_query']('', 'SELECT COUNT(*) FROM {db_prefix}themes WHERE variable = \'auto_notify\'');
+list($maxMembers) = $smcFunc['db_fetch_row']($request);
+$smcFunc['db_free_result']($request);
+
+while (!$is_done)
```

Comment:
while run at least once -> possible issue on multi execute of upgrade runs?

Reply:
I'm not following the issue with multiple runs.  The logic looks sound and we do delete the setting from the table after we are done with this step, which would prevent the issue from multiple runs.

Reply:
Should not be an issue due to the insert ignore.